### ISSUE TITLE
Using Atom to edit commit-messages shouldn't open new window

### ISSUE THREAD

**Issue by Alhadis** [atom/atom#11709]
If I use Atom as Git's editor as described [here](https://github.com/atom/language-git#git-editing-support-in-atom-), a new window opens when I run `git commit`. It should first check if the containing folder's already been opened and switch to that window.

### Expectation
<img width="746" alt="Expectation" src="https://cloud.githubusercontent.com/assets/2346707/15084249/a129b79e-1415-11e6-9094-ec3f5d52d97a.png" />

### Reality
<img width="746" alt="Reality" src="https://cloud.githubusercontent.com/assets/2346707/15084276/c4625e0a-1415-11e6-85da-28cd8fead74b.png" />

**Comment by 50Wliu** [comment]
I believe this is blocked on Atom being able to terminate from `--wait` properly if you only close a tab and not an entire window.

**Comment by damieng** [comment]
Duplicate of https://github.com/atom/atom/issues/10244
